homeassistant/components/livisi/config_flow.py

```diff
@@ -80,8 +81,6 @@ async def create_entity(self, user_input: dict[str, str]) -> FlowResult:
             title=f"SHC {controller_type}",
             data={
                 **user_input,
-                CONF_HOST: self.host,
-                "os_version": controller_info["osVersion"],
-                "config_version": controller_info["configVersion"],
+                CONF_OS_VERSION: controller_info["osVersion"],
```

Comment:
Will the OS version never change after the user configures the integration and creates the config entry? I'm asking since this value is not something the user enters but comes from the controller. Is the controller able to be updated to a different OS version?

Reply:
I removed the os version of the LIVISI controller because it doesn't bring value to Home Assistant. Thank you for your comment.